src/parser.js

```diff
@@ -93,6 +93,8 @@ EPUBJS.Parser.prototype.packageContents = function(packageXml, baseUrl){
 
 	spineIndexByURL = {};
 	spine.forEach(function(item){
+		spineIndexByURL[decodeURI(item.href)] = item.index;
+		spineIndexByURL[encodeURI(item.href)] = item.index;
```

Comment:
I think we should just always use the encodeURI of the item.href, we this can just be a single ```spineIndexByURL[encodeURI(item.href)] = item.index;```